kettle/update.py

```diff
@@ -38,31 +41,35 @@ def call(cmd):
 
 
 def main():
-    call('time python3 make_db.py --buckets buckets.yaml --junit --threads 32')
+    call('time python3 make_db.py --buckets buckets.yaml --junit --threads {THREADS}')
 
-    bq_cmd = 'bq load --source_format=NEWLINE_DELIMITED_JSON --max_bad_records=1000'
+    bq_cmd = 'bq load --source_format=NEWLINE_DELIMITED_JSON --max_bad_records={MAX_BAD_RECORDS}'
     mj_cmd = 'pypy3 make_json.py'
 
     mj_ext = ''
     bq_ext = ''
     try:
-        call(mj_cmd + ' --days 1 --assert-oldest 1.9')
+        call(mj_cmd + ' --days 1 --assert-oldest {DAYS_OLD}')
     except OSError:
         # cycle daily/weekly tables
         bq_ext = ' --replace'
         mj_ext = ' --reset-emitted'
 
-    call(mj_cmd + mj_ext + ' --days 1 | pv | gzip > build_day.json.gz')
-    call(bq_cmd + bq_ext + ' k8s-gubernator:build.day build_day.json.gz schema.json')
+    if os.getenv('DEPLOYMENT', 'prod') == "staging":
+        call(f'{mj_cmd} | pv | gzip > build_staging.json.gz')
+        call(f'{bq_cmd} k8s-gubernator:build.staging build_staging.json.gz schema.json')
+    else:
+        call(f'{mj_cmd} {mj_ext} --days 1 | pv | gzip > build_day.json.gz')
+        call(bq_cmd + bq_ext + ' k8s-gubernator:build.day build_day.json.gz schema.json')
```

Comment:
style nit, maybe stick with f strings for consistency?
```suggestion
        call(f'{bq_cmd} {bq_ext} k8s-gubernator:build.day build_day.json.gz schema.json')
```